pkg/base/linuxutils/partitionhelper/partition_helper.go

```diff
@@ -275,17 +275,17 @@ func (p *WrapPartitionImpl) GetPartitionUUID(device, partNum string) (string, er
 // SyncPartitionTable syncs partition table for specific device
 // Receives device path to sync with partprobe, device could be an empty string (sync for all devices in the system)
 // Returns error if something went wrong
-func (p *WrapPartitionImpl) SyncPartitionTable(device string) (err error) {
+func (p *WrapPartitionImpl) SyncPartitionTable(device string) (outStr string, errStr string, err error) {
```

Comment:
are you sure that we need to return both of messages? Is outStr used anywhere?

Reply:
currently we have not used the outStr,  but as SyncPartitionTable is using RunCmd (which return the stdout and stderr, err),  we'd better now make it return all.  In case for future usage.